go/test/endtoend/vtgate/reservedconn/sysvar_test.go

```diff
@@ -329,16 +329,11 @@ func TestEnableSystemSettings(t *testing.T) {
 	require.NoError(t, err)
 	defer conn.Close()
 
-	// Insert a single row to correctly select @@enable_system_settings.
-	// See: https://github.com/vitessio/vitess/issues/7301
-	checkedExec(t, conn, "delete from test")
-	checkedExec(t, conn, "insert into test (id, val1, val2, val3) values (1, null, 0, 0)")
-
 	// test set @@enable_system_settings to false and true
 	checkedExec(t, conn, "set enable_system_settings = false")
-	assertMatches(t, conn, `select @@enable_system_settings from test`, `[[INT64(0)]]`)
+	assertMatches(t, conn, `select @@enable_system_settings`, `[[INT32(0)]]`)
```

Comment:
Instead of removing the test when we send the query to the tablet, we should double up and test both against `test` and against `dual`

Reply:
Oh, and they should both use the same type - which in mysql is int24

Reply:
Roger that 👍🏻

Reply:
Fixed in 0294465 & 2d91120.